codesydney/aicta
# Update pic on first screen and redirect the navbar

## Rationale
- This is for improving the first-screen pic and navbarl' s link

## Advice for Reviewers & Testing Notes 
Changes
- I replaced the main picture for frist screen
- I remove the "wave picture" in the first-screen
- I redirected the navbar button to inner pages content
Why
-The new pic is more suitable for organization
-We need to make a one page web so I change the navbar's direction
Review
- Check the overall experience.
- Click on the navbar make sure it works

## Screenshots:
- Add a screenshot or two of your changes, where applicable.

## Task Name
- Issue Title


## Linting Checklist
